web_full_stack/python/Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flask = "==1.1.2"
+flask = "==2.2.2"
```

Comment:
A candidate reported this wasn't working. It worked for me this morning, but might as well bump it if it's reported to be causing problems.